pages.nl/common/visudo.md

```diff
@@ -1,7 +1,7 @@
 # visudo
 
 > Bewerk veilig het sudoers-bestand.
-> Meer informatie: <https://www.sudo.ws/man/1.8.13/visudo.man.html>.
+> Meer informatie: <https://www.sudo.ws/docs/man/visudo.man/>.
```

Comment:
```suggestion
> Meer informatie: <https://www.sudo.ws/docs/man/visudo.man>.
```

Reply:
@mfrw the page automatically redirects to `...man/`, so we should directly link to it.

Reply:
We have redirections in a lot of pages. I prefer without `/` at the end, but isn't an strong rule.